[Clarabel.cpp] Remove Unicode characters

Cherry-pick of https://github.com/oxfordcontrol/Clarabel.cpp/pull/29.
We can drop this Drake patch when upstream merges it.


diff --git a/include/c/DefaultInfo.h b/include/c/DefaultInfo.h
--- include/c/DefaultInfo.h
+++ include/c/DefaultInfo.h
@@ -9,7 +9,7 @@
 // ClarabelDefaultInfo types
 typedef struct ClarabelDefaultInfo_f64
 {
-    double μ;
+    double mu;
     double sigma;
     double step_length;
     uint32_t iterations;
@@ -28,7 +28,7 @@ typedef struct ClarabelDefaultInfo_f64
 
 typedef struct ClarabelDefaultInfo_f32
 {
-    float μ;
+    float mu;
     float sigma;
     float step_length;
     uint32_t iterations;
--- include/cpp/DefaultInfo.h
+++ include/cpp/DefaultInfo.h
@@ -13,7 +13,7 @@ struct DefaultInfo
 {
     static_assert(std::is_same<T, float>::value || std::is_same<T, double>::value, "T must be float or double");
 
-    T μ;
+    T mu;
     T sigma;
     T step_length;
     uint32_t iterations;
